extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/linkHandler/PeertubeChannelLinkHandlerFactory.java

```diff
@@ -31,11 +31,11 @@ public String getUrl(String id, List<String> contentFilters, String searchFilter
     @Override
     public String getUrl(String id, List<String> contentFilter, String sortFilter, String baseUrl)
             throws ParsingException {
-        return baseUrl + ACCOUNTS_ENDPOINT + id;
+        return baseUrl + API_ENDPOINT + id;
```

Comment:
Please change it to something like the following to remain backwards compatible:

```suggestion
        if (id.matches(ID_PATTERN)) {
            return baseUrl + API_ENDPOINT + id;
        } else {
            // This is needed for compatibility with older versions were we didn't support video channels yet
            return baseUrl + API_ENDPOINT + "accounts/" + id;
        }
```